src/compiler/c.rs

```diff
@@ -75,7 +75,7 @@ pub struct ParsedArguments {
     pub common_args: Vec<OsString>,
     /// Whether or not the `-showIncludes` argument is passed on MSVC
     pub msvc_show_includes: bool,
-    /// Whether the compilation is generating profiling data.
+    /// Whether the compilation is generating profiling or coverage data.
     pub profile_generate: bool,
```

Comment:
I'd rename this from `profile_generate` to `keep_line_numbers`, because it's only used to suppress the `-P` flag.

Reply:
That seems reasonable.

Reply:
Alph applo